play/src/front/Connection/LocalUserStore.ts

```diff
@@ -289,6 +290,16 @@ class LocalUserStore {
         return localStorage.getItem(chatSounds) !== "false";
     }
 
+    getFoldersOpened(): Set<string> {
+        const value = localStorage.getItem(foldersOpened);
+        if (value === null) return new Set();
+        return new Set(JSON.parse(value));
```

Comment:
Let's try to Zod and try / catch the parse. In case of error, return empty array (+ warnings)